layouts/partials/scripts.html

```diff
@@ -27,17 +27,23 @@
 </script>
 {{ end -}}
 
+<!--for the splitter-->
+<!--script src="https://unpkg.com/split.js/dist/split.min.js"></script-->
+<script src="/js/split-1.6.0.js" integrity="sha384-0blL3GqHy6+9fw0cyY2Aoiwg4onHAtslAs4OkqZY7UQBrR65/K4gI+hxLdWDrjpz"></script>
```

Comment:
Even better: move this into `hooks/body-end.html`

Reply:
Thanks for the good catch, although I think I will need to put it in `head-end.html` instead else it doesn't get recognised.